apps/change-log.md

```diff
@@ -10,6 +10,24 @@ In order to keep track of these changes we recommend you to follow [@epagesdevs]
 
 <hr>
 
+## 2016-04-11
+
+### Changes with software release 6.17.43
+
+#### <i class="fa fa-plus"></i> New
+
+* [`POST` products](https://developer.epages.com/apps/api-reference/post-shops-shopid-products.html)
```

Comment:
we should use relative links here:
*api-reference/post-shops-shopid-products.html*
instead of
*https://developer.epages.com/apps/api-reference/post-shops-shopid-products.html*